### ISSUE TITLE
Google play rejected apk claiming the TrustManager fix

### ISSUE THREAD

**Issue by DrMoriarty** [godotengine/godot#11448]
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
Godot 2.1.4-stable
MacOSX 10.12.5 (16F73)
Android 4.4.2

**Issue description:**
I built an apk and uploaded it to google play. It sent me an email with this text:

![2017-09-20 23 48 38](https://user-images.githubusercontent.com/1177068/30666845-48dc2070-9e5e-11e7-9411-aad2439ea5c2.png)

The mentioned google help article is [https://support.google.com/faqs/answer/6346016](https://support.google.com/faqs/answer/6346016)

I have found the TrustManager in `godot/platform/android/java/src/org/godotengine/godot/utils/CustomSSLSocketFactory.java` but I don't understand what should I do with it.

**Steps to reproduce:**

Build an apk with godot 2.1.4 template and upload it to google play console.

**Link to minimal example project:**

**Comment by volzhs** [comment]
where did you upload it? in alpha, beta or production?

**Comment by volzhs** [comment]
@DrMoriarty can you make and test android template with #11449 PR?

**Comment by puppetmaster-** [comment]
total offtopic....
Why is this fix date so old?
You have a very funny domain name. org.asmstudio.spacecute (remove the first dot) 😄

**Comment by volzhs** [comment]
@DrMoriarty The deadline to fix for trustmanager issue was set by google at around Feb. 2016.
so... looks old.

**Comment by DrMoriarty** [comment]
@volzhs I uploaded the build in alpha section

**Comment by volzhs** [comment]
hm. I uploaded mine in alpha recently. but I didn't get that email.
don't know why...

**Comment by DrMoriarty** [comment]
@volzhs just uploaded the build with #11449 PR
The same result:

```

Vulnerability | APK Version(s) | Deadline to fix
-- | -- | --
TrustManagerYou can find more information about TrustManager in this Google Help Center article. | 19 | 05/17/2016


```

**Comment by akien-mga** [comment]
@DrMoriarty Are you using custom Android modules, and if so which ones?

**Comment by DrMoriarty** [comment]
@volzhs @akien-mga Update:
Google sent me one more email about the previous build version (19). I checked again after several minutes and the error message disappeared from the developer console.
The PR #11449 actually fixed the problem! Thank you @volzhs

**Comment by akien-mga** [comment]
Awesome :)

@volzhs Can you do the similar PR for master?

**Comment by akien-mga** [comment]
Fixed by #11449.
